pulsar-client/src/main/java/org/apache/pulsar/client/impl/UnAckedMessageTracker.java

```diff
@@ -114,6 +121,11 @@ public UnAckedMessageTracker(PulsarClientImpl client, ConsumerBase<?> consumerBa
         ReentrantReadWriteLock readWriteLock = new ReentrantReadWriteLock();
         this.readLock = readWriteLock.readLock();
         this.writeLock = readWriteLock.writeLock();
+
+        InstrumentProvider ip = client.instrumentProvider();
+        consumerAckTimeoutsCounter = ip.newCounter("pulsar.client.consumer.message.ack.timeout", Unit.Messages,
+                "Number of ack timeouts events", consumerBase.getTopic(), Attributes.empty());
```

Comment:
The description is in "events" and unit is in messages.
I suggest: "The number of messages that were not acknowledged in the configured timeout period, hence, were requested by the client to be redelivered"